Utilities/GenerateTypescriptInterfaces/GenerateTypescriptInterfaces.csproj

```diff
@@ -8,6 +8,7 @@
 	<ItemGroup>
 	  <PackageReference Include="CaseExtensions" Version="1.0.3" />
 	  <PackageReference Include="CommandLineParser" Version="2.7.82" />
+	  <PackageReference Include="Dapper" Version="2.0.35" />
```

Comment:
I am unsure why Dapper is here, the package shouldn't need it.
I will look into removing it.